release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -113,6 +113,20 @@ io:println(nameText); // "DanBrown"
 
 #### Runtime
 
+##### JAVA Runtime API
+
+New runtime java API is introduced for creating distinct errors which supports hierarchical errors.
+
+```java
+BError createError(Module module, String errorTypeName, BString message, BError cause, Object details)
+```
+
+The runtime java API 'createDistinctError' is deprecated and recommend to use above error creation API. 
+
+##### Configurable Variables
+
+Improved error messages for configurable variables by including more information and `Config.toml` line numbers.
```

Comment:
```suggestion
Improved error messages are introduced for configurable variables by including more information and adding line numbers in the `Config.toml` file.
```

Reply:
I think what we are trying to say is that we include line number info in the error messages. Shall we say
```suggestion
Improved error messages are introduced for configurable variables by including more information and the relevant `Config.toml` line numbers.
```

Reply:
+1